src/MapViewController.swift

```diff
@@ -17,7 +17,21 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     var lastSetPoint: TGGeoPoint?
     var shouldShowCurrentLocation = false
     public var shouldFollowCurrentLocation = false
-    public var findMeButton = MapViewController.createFindMeButton()
+    public var findMeButton = UIButton(type: .Custom)
+
+    init(){
+        super.init(nibName: nil, bundle: nil)
+        defer {
+            findMeButton = createFindMeButton()
+        }
+    }
+    
+    required public init?(coder aDecoder: NSCoder) {
```

Comment:
This is needed because Apple marked `init?(coder:)` as required implementation anytime you add a convenience initializer like `init()`. No idea why but it's been a thing since iOS 9.